### PR TITLE
some little webui test fixes

### PR DESCRIPTION
**Pull request by martinpitt**
I was playing around with the new anaconda web ui, this is my "dip my toe into the water" :droplet:

### CODE REVIEW

**Comment by martinpitt** [comment]
:thinking: [rawhide rpm build](https://download.copr.fedorainfracloud.org/results/packit/rhinstaller-anaconda-4019/fedora-rawhide-x86_64/04084564-anacondam/builder-live.log.gz) failed on some autoconf/automake version problem. Is that "known"?

**Review by KKoukiou — APPROVED** [review]

File: ui/webui/test/helpers/installer.py [hunk 1]
```diff
@@ -35,6 +35,9 @@ def next(self):
 
     def open(self):
         self.browser.open("/cockpit/@localhost/anaconda-webui/index.html#/installation-language")
+        # wait until the page is sufficiently initialized
```
Comment (M4rtinK) [inline ui/webui/test/helpers/installer.py:38]: Tagging @velezd as I remember him mentioning some race-condition like behaviour when running some of the Web UI tests. Good to see it possibly fixed. :)

**Review by M4rtinK — APPROVED** [review]
Looks good to me & welcome! :)

**Pull request merged**